src/witan/models/run_models.clj

```diff
@@ -164,14 +164,14 @@
                            :proportion-male-newborns
                            (double (/ (:number-male-newborns user-parameters)
                                       (:number-all-newborns user-parameters))))
-                    (catch Exception e (println (format "Caught Error: %s"
-                                                        (.getMessage e)))))]
-    (println (format "%d option validation errors\n%s" (count errors)
-                     (clojure.string/join "\n" errors)))
-    (println (format "\nInput used: %s\nOutput used: %s\n"
-                     (:input-config (:options (parse-opts args cli-options)))
-                     (:output-projections (:options (parse-opts args cli-options)))))
-    (println (format "Preparing projection for %s... " (get-district gss-code)))
+                    (catch Exception e (info (format "Caught Error: %s"
+                                                     (.getMessage e)))))]
+    (info (format "\n%d option validation errors\n%s" (count errors)
```

Comment:
I assume these are all useful as information from the command-line tool. It's possible to use debug level for things that you don't really want to see every time you run, but only when you want to get detailed information (because something went wrong, presumably)